tensorflow/tools/ci_build/Dockerfile.rocm

```diff
@@ -73,9 +87,25 @@ ENV PATH="$OPENCL_ROOT/bin:${PATH}"
 # Add target file to help determine which device(s) to build for
 RUN bash -c 'echo -e "gfx803\ngfx900\ngfx906" >> /opt/rocm/bin/target.lst'
 
-# Workadround : build HIP from source that has HIP PR 933 + PR 936 merged
-RUN cd $HOME && git clone -b pr_933_936_fixes https://github.com/deven-amd/HIP.git
-RUN cd $HOME/HIP && mkdir build && cd build && cmake .. && make package -j$(nproc) && dpkg -i *.deb
+# Build rocBLAS from source
+RUN cd $HOME && git clone -b master-rocm-2.1 https://github.com/ROCmSoftwarePlatform/rocBLAS.git && cd rocBLAS && ./install.sh -id
+RUN cd $HOME && git clone -b master-rocm-2.1 https://github.com/ROCmSoftwarePlatform/hipBLAS.git && cd hipBLAS && ./install.sh -id
+
+# Build rcFFT from source
+RUN cd $HOME && git clone -b master-rocm-2.1 https://github.com/ROCmSoftwarePlatform/rocFFT.git && cd rocFFT && ./install.sh -icd
```

Comment:
@parallelo can we drop the build option for rocFFT clients here?

Reply:
Fine by me.  Will drop for all of the libs.